.circleci/config.yml

```diff
@@ -31,6 +31,9 @@ jobs:
       - run: cargo audit
       - run: cargo build --release
       - run: cargo test --release
+      - run: |
+          cargo install --force cargo-cache
```

Comment:
Why force the reinstall of the package ?

Reply:
Because cargo is a sensitive little one and he whines if I try to install and it's already installed :
"error: binary `cargo-cache` already exists in destination" as you can see here : https://app.circleci.com/pipelines/github/OpenBAS-Platform/agent/666/workflows/0d07277e-7798-44df-af34-afffa30c610c/jobs/2800

Reply:
I'm not too happy about reinstalling every time either but I'm unsure of a better solution ...

Reply:
I improved things by removing the --force on all the build except for macos_x86_64. For some reason, this is the only build for which cargo install ends up with a failure if the package already exists. Even forcing the old option of upgrading if it already exists fails as you can see here : https://app.circleci.com/pipelines/github/OpenBAS-Platform/agent/670/workflows/e11c32ee-2a30-450a-b448-eae1850de87a/jobs/2832

Reply:
Thank you very much for digging